frontend/src/Users/Users.tsx

```diff
@@ -1,5 +1,173 @@
+import { Box, Checkbox, FormControlLabel, FormGroup, InputLabel, MenuItem, Select } from "@mui/material";
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import { useTranslation } from "react-i18next";
+
+interface EditUserProps {
+  open: boolean;
+  name: string;
+  closeDialog: () => void;
+}
+
+function EditUserDialog({ open, name, closeDialog }: EditUserProps) {
+  const { t } = useTranslation();
+  const [role, setRole] = React.useState('guest');
+  return (
+    <Dialog open={open} onClose={closeDialog}>
+      <DialogTitle>{t('users.dialog.title')}</DialogTitle>
+      <DialogContent>
+        <TextField
+          autoFocus
+          margin="dense"
+          id="username"
+          label="Username"
+          type="text"
+          fullWidth
+          variant="standard"
+          placeholder="Username"
+        />
+        <InputLabel id="role-label" sx={{display: 'inline-block', marginRight: '10px'}}>Role</InputLabel>
+        <Select
+          labelId="role-label"
+          id="demo-simple-select"
+          value={role}
+          label={t('users.role')}
+          onChange={(event) => {setRole(event.target.value)}}
+        >
+          <MenuItem value="editor">{t('users.dialog.editor')}</MenuItem>
+          <MenuItem value="intern">{t('users.dialog.intern')}</MenuItem>
+          <MenuItem value="guest">{t('users.dialog.guest')}</MenuItem>
+        </Select>
+        <FormGroup>
+        <FormControlLabel
+          control={<Checkbox name="isBlocked" />}
+          label={t('users.dialog.block-access')}
+        />
+        </FormGroup>
+        <TextField
+          autoFocus
+          margin="dense"
+          id="fullName"
+          label={t('users.full-name')}
+          type="text"
+          fullWidth
+          variant="standard"
+          value={name}
+          placeholder={t('users.full-name')!!}
+        />
+        <TextField
```

Comment:
I think that admin needs ability to see password. May be you can even show password always

Reply:
Our backend dudes store password as hash, so this is not possible :sob: :sob: :sob: :sob: :sob: :sob: :sob: :sob: